scripts/removed-icons.js

```diff
@@ -0,0 +1,314 @@
+module.exports = [
```

Comment:
We need a way to update this file automatically. Do you have something in mind about it @jorgeamadosoria?

Reply:
No. I don't really mind updating this manually. We don't remove that many icons, this is easy to keep updated just by reading the Release notifications and getting the removed issues from there.
I'm fine with this setup.

Reply:
I think that it should be inside the "Auto Release" workflow, after [the step that bumps simple-icons version](https://github.com/simple-icons/simple-icons-website/blob/45979f1e6fe1489e6d94d7c6b36cc2b06f627cec/.github/workflows/auto-release.yml#L27-L31).

> No. I don't really mind updating this manually.

If so, you maybe need to create a release checklist and declare some compromise to maintain it. Anyways, please, note that we don't want to depend on a single contributor for periodical manual tasks. If you left the organization tomorrow for whatever reason someone else would be obliged to keep this, so I am totally against not automating it.

Reply:
yeah, no I get that. Not sure how to add this to the Auto Release workflow. I'll take a look.